### PR TITLE
feat: login redirect

### PR DESCRIPTION
**Pull request by vivianamck**
## Issue ticket number and link

https://trello.com/c/C4DqafFX

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [X] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- Login redirect to events cards

## What is the new behavior?

- Added login redirect by user profile
- Admin redirect to events list view(create, edit and remove) and others redirect to events cards view

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: src/components/Login/Login.tsx [hunk 1]
```diff
@@ -133,10 +135,21 @@ export default function Login({ onLogin, loading }: LoginProps): JSX.Element {
...
 
+  useEffect(() => {
+    const userState = user.state
+    const shouldRedirectTo = userState.username
+      ? userState.isAdmin
+        ? `/events/list`
+        : `/`
+      : eventId
+      ? `/event-info/${eventId}`
+      : `/login`
```
Comment (jlora-mck) [inline src/components/Login/Login.tsx:149]: Please refactor this in a way that is understandable, nested ternary are hard to ready and hard to mantain

Comment (vivianamck) [reply src/components/Login/Login.tsx]: Done

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review comment

File: src/components/Login/Login.tsx [hunk 2]
```diff
@@ -133,10 +135,20 @@ export default function Login({ onLogin, loading }: LoginProps): JSX.Element {
...
 
   const handleLoginClicked = () => {
-    const redirectURL = eventId ? `/event-info/${eventId}` : `/`
-    onLogin(userName, password, () => history.push(redirectURL))
+    onLogin(userName, password)
   }
 
+  useEffect(() => {
+    const userState = user.state
+    let shouldRedirectTo = '/login'
```
[the remark this answers is not in the document]
Comment (vivianamck) [reply src/components/Login/Login.tsx:143]: Done

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Review by Sofiag8 — APPROVED** [review]

**Pull request merged**